### PR TITLE
MBS-6138: Add alert for hidden activities

### PR DESCRIPTION
**Pull request by sh-csg**

### CODE REVIEW

File: templates/activityselector.mustache [hunk 1]
```diff
@@ -71,11 +71,12 @@
         <option value="{{id}}" disabled class="learningmap-disabled-activity" title="{{#str}} nocompletionenabled, learningmap {{/str}}">{{#shortentext}} 30, {{{name}}} {{/shortentext}}</option>
         {{/completionenabled}}
         {{#completionenabled}}
-        <option value="{{id}}" title="{{name}}">{{#shortentext}} 30, {{{name}}} {{/shortentext}}</option>
+        <option value="{{id}}" title="{{name}}" data-activity-hidden="{{hidden}}">{{#shortentext}} 30, {{{name}}} {{/shortentext}}</option>
```
Comment (fhuebler) [inline templates/activityselector.mustache:74]: Please add the placeholder tag "hidden" above in the json example context

File: amd/src/learningmap.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import {exception as displayException} from 'core/notification';
 import Templates from 'core/templates';
 import placestore from 'mod_learningmap/placestore';
-
+/* eslint-disable */
```
Comment (fhuebler) [inline amd/src/learningmap.js:4]: leftover?

**Comment by fhuebler** [comment]
Looks good, I tested manually with firefox

File: amd/src/learningmap.js (hunk 2 above)
Comment (sh-csg) [reply amd/src/learningmap.js]: Yes ;-)

File: templates/activityselector.mustache (hunk 1 above)
Comment (sh-csg) [reply templates/activityselector.mustache:77]: Fixed!

**Pull request merged**